Assets/_BForBoss/_Utility/Scripts/ObjectPooler.cs

```diff
@@ -37,19 +37,22 @@ public ObjectPooler(string poolName, Func<T> buildObject, Action<T> actionOnGet
 			this.buildObject = buildObject;
 			this.actionOnGet = actionOnGet;
 			this.actionOnRelease = actionOnRelease;
+			
+			SceneManager.activeSceneChanged += OnActiveSceneChanged;
```

Comment:
Does this also have to be wrapped in a $if statement?

Reply:
Yeah it does, good catch